app/hedge.py

```diff
@@ -115,7 +118,7 @@ def cleanup():
 #  =======================================
 # Main Loop
 #  =======================================
-def main(last_ran, box_id, cycle_time):
+def main(last_ran, box_id, cycle_time, PIZERO_IP, PIZERO_FTP_USERNAME, PIZERO_FTP_PASSWORD):
```

Comment:
You don't need to pass these constants around between functions as they are defined at the module scope.

Reply:
Yeah that's what I thought but I got a load of issues with it, I think it might be because we're using python3.5